src/commands/gen-js/library.ts

```diff
@@ -137,12 +97,5 @@ export async function genJS(
     }
   })
 
-  return Promise.resolve(prettier.format(outputText, { parser: 'babylon' }))
+  return prettier.format(outputText, { parser: 'babylon' })
```

Comment:
Good catch; don't actually recall why this function was `async` in the first place 🤔